client/app/sessions/sessions.js

```diff
@@ -42,7 +40,7 @@ angular.module('dinnerDaddy.sessions', [])
     .then(function(session) {
       console.log('created session');
       $rootScope.currentSession = session;
-      Socket.emit('session', {sessionName: session.sessionName});
+      Socket.emit('session', {sessionID: session.id});
```

Comment:
can we camelCase id throughout? So sessionID would be sessionId and userID would be userId. That's the convention we're mostly using elsewhere.